velox/vector/DictionaryVector.h

```diff
@@ -152,7 +152,20 @@ class DictionaryVector : public SimpleVector<T> {
   }
 
   BaseVector* loadedVector() override {
```

Comment:
Another nit: loadedVector() seems to me like its asking a question : 'Is the vector loaded ?' , where infact this actually loads a vector , maybe we should change the name to loadVector ?

Reply:
By convention, getters do not have a 'get' prefix. loadedVector() returns / gets a vector that has been loaded. A bool check would be named isLoadedVector.

Reply:
Makes sense. Since this would be a refactoring in BaseVector and subclasses, I suggest to do this in a separate PR.